### PR TITLE
Feature/divya week3

### PR DESCRIPTION
**Pull request by 890773**
Week-3: Threads, Lambdas and streams, Exceptions

### CODE REVIEW

File: week3/Exceptions/CopyingFiles.java [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+public class CopyingFiles {
+    private static final int BUFFER_SIZE = 4096;
+
+    public static void main(String[] args) {
+        Scanner sc = new Scanner(System.in);
+        String inputFile = sc.nextLine();
+        String outputFile = sc.nextLine();
+        File input = new File(inputFile);
+        File output = new File(outputFile);
+
```
Comment (VBhaskarachary) [inline week3/Exceptions/CopyingFiles.java:15]: As per requirement to give  input and output file path direct

File: week3/Exceptions/DivideByZeroAndNullException.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    public static void main(String[] args) {
+        DivideByZeroAndNullException obj = new DivideByZeroAndNullException();
+        System.out.println("num1=");
+        int num1 = obj.getInt();
+        System.out.println("num2=");
+        int num2 = obj.getInt();
+        System.out.println("enter string");
+        String s = obj.getString();
+        try {
+            int result = num1 / num2;
```
Comment (VBhaskarachary) [inline week3/Exceptions/DivideByZeroAndNullException.java]: Pls dont use system input for string,

**Review by VBhaskarachary — APPROVED** [review]
changes looks good

File: week3/Exceptions/DivideByZeroAndNullException.java (hunk 2 above)
Comment (890773) [reply week3/Exceptions/DivideByZeroAndNullException.java]: Fixed.

File: week3/Exceptions/CopyingFiles.java (hunk 1 above)
Comment (890773) [reply week3/Exceptions/CopyingFiles.java:15]: Fixed

**Pull request merged**